src/viam/components/base/client.py

```diff
@@ -107,13 +109,18 @@ async def stop(
         request = StopRequest(name=self.name, extra=dict_to_struct(extra))
         await self.client.Stop(request, timeout=timeout)
 
-    async def is_moving(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None) -> bool:
-        if extra is None:
-            extra = {}
+    async def is_moving(self, *, timeout: Optional[float] = None) -> bool:
```

Comment:
What happened to the `extra` parameter here?

Reply:
I noticed that `IsMovingRequest` doesn't actually take `extra`, so I deleted it.

Reply:
Ah interesting, yeah I thought all gRPC messages had an `extra` field for some reason, but looks like you have to explicitly define one.

Reply:
This is hypothetically backward-breaking though, what are your thoughts on removing this @njooma?

Reply:
@benjirewis Naveed has agreed that removing it would be the way to go.